sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -417,6 +422,24 @@ Exceptions:
 * Throws with error code `invalid` if `nodeEvent` is not associated with a
   graph node.
 
+|
+[source,c++]
+----
+event get_external_event();
+----
+|Gets the external event associated with this node. Can only be called for nodes
+with type `node_type::external_event`.
```

Comment:
how about other member functions if the node type is node_type::external_event

shall we add something elsewhere for spec completeness?

Reply:
Other member functions are valid for `node_type::external_event` unless otherwise noted, for example the `update_range/nd_range()` member functions do say this under exceptions:

``Throws with error code `invalid` if the type of the node is not a kernel
  execution.``